### PR TITLE
What is new 0.2.0

### PR DESCRIPTION
**Pull request by erpenbach**

### CODE REVIEW

**Comment by erpenbach** [comment]
Created What's New in 0.2.0 document

File: publish/2019-10-20-WhatsNew020.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Control
+The Kabanero operator introduces the `--targetNameSpace` parameter, which defines the set of valid NameSpaces for which the Appsody operator watches for the deployment of applications.  
+
+Ease of Use
+The Kabanero operator has introduced variable substitution in the collection pipelines, simplifying collection curation.   The Kabanero Foundation provides a must-gather script to help diagnostic activities, and an uninstall script which can help in try-it and POC activities.
+
+Kabanero installs an OKD console extension we've called the OKD landing page.   The Landing Page has been enhanced to display active collections, links to supporting software, like the Tekton dashboard, as well as delivering documentation and new Guides accessible right from your OKD cluster.
+
+image::/img/guide/Kabanero020.png[link="/img/Kabanero020.png" alt="Kabanero 0.2.0"]
```
Comment (alohr51) [inline publish/2019-10-20-WhatsNew020.adoc]: this is the first img we're adding to blogs, so the below image file should go in a folder `img` at the root of the repo. Then both links would be `/img/blog/Kabanero020.png`

**Review by alohr51 — CHANGES_REQUESTED** [review]

**Comment by alohr51** [comment]
we should make sure the content aligns with Nate's update to the what's new doc in this PR: https://github.com/kabanero-io/docs/pull/148

**Comment by alohr51** [comment]
should remove the img at `publish/img/Kabanero020.png ` looks like both are in the PR now.

**Comment by alohr51** [comment]
and the link needs to change as described in https://github.com/kabanero-io/blogs/pull/22/files#r336788961

File: publish/2019-10-20-WhatsNew020.adoc (hunk 1 above)
Comment (erpenbach) [reply publish/2019-10-20-WhatsNew020.adoc]: I've updated to put the png file in /img/blog/

File: publish/2019-10-20-WhatsNew020.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "What's New in Release 0.2.0"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/50876789
+author_github: https://github.com/stephenkinder
+seo-title: What is new in release 0.2.0
+seo-description: What is new in the 0.2.0 release - this covers the new namespace support, ease of use enhancements and guides.
```
Comment (nateziemann) [inline publish/2019-10-20-WhatsNew020.adoc]: seo-description:  See what’s hot in Kabanero v0.2.0. New features include more customization for enterprise, managing microservice composed applications, and Guides.

File: publish/2019-10-20-WhatsNew020.adoc [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+blog_description: "What is new in the 0.2.0 release - this covers the new namespace support, ease of use enhancements and guides."
```
Comment (nateziemann) [inline publish/2019-10-20-WhatsNew020.adoc]: blog_description: See what’s hot in Kabanero v0.2.0. New features include more customization for enterprise, managing microservice composed applications, and Guides.

**Review by alohr51 — APPROVED** [review]

**Pull request merged**